Resources/Prototypes/Entities/Mobs/Species/moth.yml

```
@@ -138,3 +138,53 @@
   components:
   - type: HumanoidAppearance
     species: Moth
+  - type: Sprite
+    layers:
+      - map: [ "enum.HumanoidVisualLayers.Chest" ]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)